### PR TITLE
querier: reformat some wide comments

### PR DESCRIPTION
**Pull request by bboreham**
I find it useful to be able to read comments in side-by-side diff view.

#### Checklist

- NA Tests updated
- NA Documentation added
- NA `CHANGELOG.md` updated

### CODE REVIEW

File: pkg/querier/blocks_store_queryable.go [hunk 1]
```diff
@@ -626,14 +627,18 @@ func filterBlocksByShard(blocks bucketindex.Blocks, queryShardIndex, queryShardC
...
-// In other words, if this function returns false, block with given compactorShardID doesn't need to be searched for series from given query shard.
+// In other words, if this function returns false, block with given compactorShardID
+// doesn't need to be searched for series from given query shard.
 //
-// In addition this function also returns whether query and compactor shard counts were divisible by each other (one way or the other).
-func canBlockWithCompactorShardIndexContainQueryShard(queryShardIndex, queryShardCount, compactorShardIndex, compactorShardCount uint64) (result bool, divisibleShardCounts bool) {
+// In addition this function also returns whether query and compactor shard counts were
+// divisible by each other (one way or the other).
+func canBlockWithCompactorShardIndexContainQueryShard(queryShardIndex, queryShardCount, compactorShardIndex,
+	compactorShardCount uint64) (result bool, divisibleShardCounts bool) {
```
Comment (colega) [inline pkg/querier/blocks_store_queryable.go:641]: I don't find this being the prettiest formatting, but we don't have a linter for it so... :shrug:

**Review by colega — APPROVED** [review]

**Comment by replay** [comment]
I think as long as there is no linter to enforce a line width, formatting comments bunch by bunch is a sisyphean task, sooner or later someone is going to undo it again. OTH I have no reason to block this PR.

File: pkg/querier/blocks_store_queryable.go (hunk 1 above)
Comment (56quarters) [reply pkg/querier/blocks_store_queryable.go:641]: We experimented with having a lint for this in enterprise metrics and it was:

* Very destructive in terms of `git blame`
* Annoying

**Comment by pracucci** [comment]
I think it's a matter of personal preference. I work on a relatively large screen and the current longer lines are not an issue for me.

**Pull request merged**